### DISCUSSION TITLE
Add support for date / time only fields in addition to `timestamp`

### DISCUSSION THREAD

**Discussion by fkrauthan** [keystonejs/keystone#7651] [Feature requests]
It would be great if there could be support for date only and time only fields. I don't always need a full timestamp. For example:

- Due Date
- Birthday
- Opening Time
- Closing Time

**Comment by Achisingh** [comment]
We are working on a **date only** field at the moment: https://github.com/keystonejs/keystone/pull/7658. For **time only** we haven't had many requests, but it is an interesting idea. Maybe others in the community can upvote this discussion, and/or provide more examples of why it would be useful for them.

**Reply by fkrauthan** [reply]
Oh nice. Subscribed to that ticket. I personally don't really need the time field as much as the date field. But wanted to mention it for completion. Guess a time field you can easily build with just a text field and some pattern validation.
